tools/hooks/requirements_changed.rb

```diff
@@ -4,7 +4,7 @@
 # of these are currently used, but are included for future-proofing.
 REQUIREMENTS = {
   "requirements.txt" => "pip install -r requirements.txt",
-  "package.json" => "npm install",
+  "package.json" => "npm prune && npm update",
```

Comment:
Yeah, it's helper text after `git pull` that detects whether package.json changed and prints a note.